packages/dds/tree/src/test/domains/json/jsDirectObject.bench.ts

```diff
@@ -52,34 +50,32 @@ export function jsObjectBench(
 
 		benchmark({
 			type: BenchmarkType.Measurement,
-			title: `averageTwoValues JS Object: '${name}'`,
+			title: `averageValues JS Object: '${name}'`,
 			before: () => {},
 			benchmarkFn: () => {
-				averageTwoValues(json, dataConsumer);
+				averageValues(json, dataConsumer);
 			},
 		});
 	}
 }
 
 function extractCoordinatesFromCanadaDirect(
 	directObj: Canada,
-	calculate: (x: number, y: number) => void,
+	calculate: (x: number) => void,
```

Comment:
Nit: since we declare `x` and `y` below, maybe `value` for this parameter?

Reply:
```suggestion
	calculate: (value: number) => void,
```